v0/destinations/woopra/data/WoopraIdentifyConfig.json

```diff
@@ -0,0 +1,282 @@
+        "metadata": {
+            "strictMultiMap": [
+                {
+                    "sourceVal": [
+                        "Individual Contributor",
+                        "individual contributor",
+                        "INDIVIDUAL CONTRIBUTOR"
+                    ],
+                    "destVal": "Individual Contributor"
+                },
+                {
+                    "sourceVal": [
+                        "Manager",
+                        "manager",
+                        "MANAGER"
+                    ],
+                    "destVal": "Manager"
+                },
+                {
+                    "sourceVal": [
+                        "Director",
+                        "director",
+                        "DIRECTOR"
+                    ],
+                    "destVal": "Director"
+                },
+                {
+                    "sourceVal": [
+                        "Executive",
+                        "executive",
+                        "EXECUTIVE"
+                    ],
+                    "destVal": "Executive"
+                },
+                {
+                    "sourceVal": [
+                        "Consultant",
+                        "consultant",
+                        "CONSULTANT"
+                    ],
+                    "destVal": "Consultant"
+                }
+            ]
+        }
+    },
+    {
+        "destKey": "cv_subscription_status",
+        "sourceKeys": [
+            "traits.subscriptionStatus",
+            "context.traits.subscriptionStatus"
+        ],
+        "required": false,
+        "metadata": {
+            "strictMultiMap": [
+                {
+                    "sourceVal": [
+                        "New",
+                        "new",
+                        "NEW"
+                    ],
+                    "destVal": "New"
+                },
+                {
+                    "sourceVal": [
+                        "Existing",
+                        "existing",
+                        "EXISTING"
+                    ],
+                    "destVal": "Existing"
+                }
+            ]
+        }
+    },
+    {
+        "destKey": "cv_prospect_or_customer",
+        "sourceKeys": [
+            "traits.prospectOrCustomer",
+            "context.traits.prospectOrCustomer"
+        ],
+        "required": false,
+        "metadata": {
+            "strictMultiMap": [
+                {
+                    "sourceVal": [
+                        "prospect",
+                        "Prospect",
+                        "PROSPECT"
+                    ],
+                    "destVal": "Prospect"
+                },
+                {
+                    "sourceVal": [
+                        "customer",
+                        "Customer",
+                        "CUSTOMER"
+                    ],
+                    "destVal": "Customer"
+                }
+            ]
+        }
+    },
```

Comment:
This looks like inspired from Mautic. 1) Woopra has not explicitly mentioned this fields, 2) we are using strict multimap, which means it is supposed to throw error if any other value is sent. As woopra doesn't have any ENUMS for these fields, it doesn't make sense to restrict the values. Lets remove these from here. and take the traits object and convert every key to cv_key-name and send out.